bitnami/mongodb-sharded/values-production.yaml

```diff
@@ -186,7 +186,7 @@ shardsvr:
     ##  - name: KIBANA_ELASTICSEARCH_URL
     ##    value: test
     ##
-    extraEnvVars:
+    extraEnvVars: []
```

Comment:
Please, make sure that we have an `if` like

```yaml
        {{- if $.Values.common.podAnnotations }}
           {{- include "common.tplvalues.render" ( dict "value" $.Values.common.podAnnotations "context" $ ) | nindent 8 }}
        {{- end }}
```
For every value changed, if not it could be rendered wrong 

```yaml
annotations:
  []
```

Reply:
I have reviewed reference in templates for all changed values, they all have either `if` or `with` to check if the input is present.